modules/admission-webhook-overview.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-overview_{context}"]
+= Admission webhooks
+
+In addition to {product-title} default admission controller plug-ins, admission webhooks enable you to introduce admission plug-ins invoked from webhook servers over HTTP at defined endpoints.
+
+There are two types of configurable admission webhooks in {product-title}:
+
+//Future xref - * During the admission process, a mutating admission controller can invoke xref:../architecture/admission-controllers.adoc#mutating-admission-webhooks[mutating admission webhooks] to perform tasks, such as injecting affinity labels.
+* During the admission process, a mutating admission controller can invoke mutating admission webhooks to perform tasks, such as injecting affinity labels.
+
+//Future xref - * At the end of the admissions process, a validating admission controller can invoke xref:../architecture/admission-controllers.adoc#validating-admission-webhooks[validating admission webhooks] to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
+* At the end of the admissions process, a validating admission controller can invoke validating admission webhooks to make sure an object is configured properly, such as validating affinity labels. If the validation passes, {product-title} schedules the object as configured.
```

Comment:
Is it "admission process" or "admissions process"? Just be consistent between the two descriptions.

Reply:
"to make sure" -> "to make sure that" or "to ensure that"

Reply:
I would say it should be 'admission process'. I have resolved this throughout.